db/Dockerfile

```diff
@@ -1,37 +0,0 @@
-FROM mvpstudio/base:v1
```

Comment:
I'm not sure we want to get rid of this. Wasn't this to build a Postgres image that we can run so y'all have a DB?